build-scripts/ubuntu-1804/build-3rd-parties.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+OUTPUT_PATH=${1:-.}
+
+function build_rocksdb_deb {
+    VERSION=$1
+    VERSION_TAG="rocksdb-$VERSION"
+
+    git clone https://github.com/evernym/rocksdb.git /tmp/rocksdb
+    cd /tmp/rocksdb
+    git checkout $VERSION_TAG
+    sed -i 's/-m rocksdb@fb.com/-m "Hyperledger <hyperledger-indy@lists.hyperledger.org>"/g' \
+        ./build_tools/make_package.sh
+    PORTABLE=1 EXTRA_CFLAGS="-fPIC" EXTRA_CXXFLAGS="-fPIC" ./build_tools/make_package.sh $VERSION
+    cp ./package/rocksdb_${VERSION}_amd64.deb $OUTPUT_PATH
+    # Install it in the system as it is needed by python-rocksdb.
+    make install
+    cd -
+    rm -rf /tmp/rocksdb
+}
+
+function build_from_pypi {
```

Comment:
Did you test this debs on ubuntu 18?

Reply:
I currently do not have a machine with 18.04. Looking to run this on the ci/cd